core/src/main/java/io/seata/core/rpc/DefaultServerMessageListenerImpl.java

```diff
@@ -106,20 +106,25 @@ public void onTrxMessage(RpcMessage request, ChannelHandlerContext ctx) {
     @Override
     public void onRegRmMessage(RpcMessage request, ChannelHandlerContext ctx, RegisterCheckAuthHandler checkAuthHandler) {
         RegisterRMRequest message = (RegisterRMRequest)request.getBody();
+        String ipAndPort = NetUtil.toStringAddress(ctx.channel().remoteAddress());
         boolean isSuccess = false;
         try {
             if (null == checkAuthHandler || checkAuthHandler.regResourceManagerCheckAuth(message)) {
                 ChannelManager.registerRMChannel(message, ctx.channel());
                 Version.putChannelVersion(ctx.channel(), message.getVersion());
                 isSuccess = true;
+                if (LOGGER.isDebugEnabled()) {
+                    LOGGER.debug("checkAuth for client:{},vgroup:{},applicationId:{}",
+                            ipAndPort,message.getTransactionServiceGroup(),message.getApplicationId());
+                }
             }
         } catch (Exception exx) {
             isSuccess = false;
             LOGGER.error(exx.getMessage());
         }
         getServerMessageSender().sendResponse(request, ctx.channel(), new RegisterRMResponse(isSuccess));
         if (LOGGER.isInfoEnabled()) {
-            LOGGER.info("rm register success,message:{},channel:{}", message, ctx.channel());
+            LOGGER.info("RM register success,message:{},channel:{}", message, ctx.channel());
```

Comment:
not recommended to deserialize the entire object of the message

Reply:
> not recommended to deserialize the entire object of the message




`Rm register success,message:RegisterRMRequest{resourceIds='jdbc:mysql://xx.xx.xx.xx/seata-account', applicationId='account-server', transactionServiceGroup='default'},channel:[id: 0xab3135a8, L:/192.168.158.80:8091 - R:/192.168.158.80:10720]`
This is the log.
I will optimize the message object ,and do we need the ctx.channel()?

Reply:
need ctx.channel()